compile-android.sh

```
@@ -33,5 +33,6 @@ for archtargetstr in \
         cargo build -p aw-server --target $target --lib $($RELEASE && echo '--release')
     else
         echo "Couldn't find directory for target $arch"
+        exit 1
```

Prevents silently failing builds